phys2bids/phys2bids.py

```diff
@@ -377,6 +377,16 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
                 phys_out[key].timeseries.insert(0, np.linspace(phys_in[run].timeseries[0][0],
                                                 phys_in[run].timeseries[0][-1],
                                                 num=phys_out[key].timeseries[0].shape[0]))
+            # Add trigger channel in the proper frequency.
+            if uniq_freq != phys_in[run].freq[chtrig]:
+                phys_out[key].ch_name.insert(1, phys_in[run].ch_name[chtrig])
+                phys_out[key].units.insert(1, phys_in[run].units[chtrig])
+                resampled_time = np.linspace(phys_in[run].timeseries[0][0],
```

Comment:
I'm probably missing something, but isn't `resampled_time` exactly the same as `phys_out[key].timeseries[0]` at this point in the code?

If the frequency the loop is in is the same as the one of the "time" channel, then `phys_out[key].timeseries[0] == phys_in[key].timeseries[0]`, and this linspace creates a perfect copy of `phys_out[key].timeseries[0]`.
If the frequency the loop is in is not the same, then lines 384-386 are the same as the second argument in `insert` in lines 377-379.

If so, wouldn't it be better to use `phys_out[key].timeseries[0]` straight away rather than recomputing it not more than five lines later?

I guess that then these three lines can be skipped, and line 387 can become `phys_out[key].timeseries.insert(1, np.interp(phys_out[key].timeseries[0], `

Reply:
You are completely correct, changes on the way